roles/ovirt-provider-ovn-driver/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+- block:
+  - name: Install ovirt-provider-ovn-driver
+    yum:
+        name: ovirt-provider-ovn-driver
+        state: latest
+
+  - name: Ensure ovn-controller is started
+    service:
+        name: ovn-controller
+        state: started
+        enabled: yes
+
+  - name: Ensure Open vSwitch is started
+    service:
+        name: openvswitch
+        state: started
+        enabled: yes
+
+  - name: Configure OVN for oVirt
+    command: >
+        vdsm-tool ovn-config {{ ovn_central }} {{ ovn_tunneling_network }}
```

Comment:
Isn't VDSM restart required after executing it?

Reply:
No, https://github.com/oVirt/ovirt-provider-ovn/blob/master/driver/scripts/setup_ovn_controller.sh is triggered via https://github.com/oVirt/ovirt-provider-ovn/blob/master/driver/vdsm_tool/ovn_config.py , which cares internally for required restarts.